helm/metac/templates/role.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "metac.fullname" . }}
+rules:
+- apiGroups:
+  - "*"
+  resources:
+  - "*"
+  verbs:
+  - "*"
```

Comment:
Do you think it will be good to expose apiGroups, resources & verbs as helm values? Since this default elevates Metac to have super power controls.

Reply:
Makes sense,  I'll make it configurable.